src/gfn/modules.py

```diff
@@ -193,6 +199,56 @@ def to_probability_distribution(
 
             return UnsqueezedCategorical(probs=probs)
 
-        # LogEdgeFlows are greedy, as are more P_B.
+        # LogEdgeFlows are greedy, as are most P_B.
         else:
             return UnsqueezedCategorical(logits=logits)
+
+
+class ConditionalDiscretePolicyEstimator(DiscretePolicyEstimator):
+    r"""Container for forward and backward policy estimators for discrete environments.
+
+    $s \mapsto (P_F(s' \mid s, c))_{s' \in Children(s)}$.
+
+    or
+
+    $s \mapsto (P_B(s' \mid s, c))_{s' \in Parents(s)}$.
```

Comment:
might be worth mentioning that this is a s very specific conditioning use-case, where the condition is encoded separately, and embeddings are concatenated.

I don't think we can do a generic one, but this should be enough as an example !

Reply:
What other conditioning approaches would be worth including? Cross attention?

Reply:
In general I would think the conditioning should be embedded / encoded separately --- or would the conditioning just need to be concatenated to the state before input? I could add support for that.

Reply:
I don't think there is an exhaustive list of ways we can process the condition. What you have is great as an example. I suggest you just add a comment or doc that the user might want to write their own module